### PR TITLE
fix(credentials): remove quality credential

### PR DESCRIPTION
**Pull request by Phil91**
## Description

Remove the quality credential

## Why

The credential is no longer valid

## Issue

Refs: #95

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/ssi-credential-issuer/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added copyright and license headers, footers (for .md files) or files (for images)

### CODE REVIEW

File: src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.Designer.cs [hunk 1]
```diff
@@ -0,0 +1,1611 @@
+﻿// <auto-generated />
```
Comment (evegufy) [inline src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.Designer.cs:1]: Could you please still add a file header?

File: src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
+using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (evegufy) [inline src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.cs:1]: Could you please still add a file header?

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.cs (hunk 2 above)
Comment (Phil91) [reply src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.cs:20]: added

File: src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.Designer.cs (hunk 1 above)
Comment (Phil91) [reply src/database/SsiCredentialIssuer.Migrations/Migrations/20240502065751_95-removeQualityCredential.Designer.cs]: added

**Review by evegufy — APPROVED** [review]

**Pull request merged**